docs/api/permissions.rst

```diff
@@ -0,0 +1,521 @@
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length
+    Connection: keep-alive
+    Content-Length: 247
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 14:54:58 GMT
+    ETag: "1437058498218"
+    Last-Modified: Thu, 16 Jul 2015 14:54:58 GMT
+    Server: nginx/1.4.6 (Ubuntu)
+
+    {
+        "data": {
+            "id": "writers", 
+            "last_modified": 1437058498218, 
+            "members": [
+                "basicauth:df93ca0ecaeaa3126595f6785b39c408be2539173c991a7b2e3181a9826a69bc"
+            ]
+        }, 
+        "permissions": {
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+Then we can give the write ACL on the bucket for the group.
+
+.. code-block::
+
+    $ echo '{"permissions": {"write": ["/buckets/servicedenuages-blog/groups/writers"]}}' | \
+        http PATCH https://kinto.dev.mozaws.net/v1/buckets/servicedenuages-blog \
+        --auth user:pass
+
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length
+    Connection: keep-alive
+    Content-Length: 254
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 14:56:55 GMT
+    ETag: "1437056437581"
+    Last-Modified: Thu, 16 Jul 2015 14:20:37 GMT
+    Server: nginx/1.4.6 (Ubuntu)
+
+    {
+        "data": {
+            "id": "servicedenuages-blog", 
+            "last_modified": 1437056437581
+        }, 
+        "permissions": {
+            "read": [
+                "system.Authenticated"
+            ], 
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6", 
+                "/buckets/servicedenuages-blog/groups/writers"
+            ]
+        }
+    }
+
+Now the user Natim can create articles.
+
+.. code-block::
+
+    $ echo '{"data":{"title": "Natim article", "content": "natims content", "published_at": "Thu Jul 16 16:59:16 CEST 2015"}}' | \
+        http POST https://kinto.dev.mozaws.net/v1/buckets/servicedenuages-blog/collections/articles/records \
+        --auth natim:
+    HTTP/1.1 201 Created
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length
+    Connection: keep-alive
+    Content-Length: 285
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 14:58:47 GMT
+    Server: nginx/1.4.6 (Ubuntu)
+
+{
+    "data": {
+        "content": "natims content", 
+        "id": "f9a61750-f61f-402b-8785-1647c9325a5d", 
+        "last_modified": 1437058727907, 
+        "published_at": "Thu Jul 16 16:59:16 CEST 2015", 
+        "title": "Natim article"
+    }, 
+    "permissions": {
+        "write": [
+            "basicauth:df93ca0ecaeaa3126595f6785b39c408be2539173c991a7b2e3181a9826a69bc"
+        ]
+    }
+}
+
+
+Listing shared items
+====================
+
+One can fetch the list of article.
```

Comment:
articles


Could you be more explicit here ? Does it rely on https://github.com/mozilla-services/cliquet/issues/354 ?